Description: Ensures the suffix list file is included in the jar
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/httpclient/pom.xml
+++ b/httpclient/pom.xml
@@ -73,6 +73,7 @@
         <filtering>true</filtering>
         <includes>
             <include>**/*.properties</include>
+            <include>**/*.txt</include>
         </includes>
       </resource>
     </resources>
